roachtest: add acceptance/build-info

petermattis opened the pull request:
Move the build-info acceptance test to a new acceptance/build-info
roachtest. Move the existing cli/node-status to
acceptance/cli/node-status. The acceptance roachtests are all going to
share the same cluster to amortize the cluster setup/teardown time.

Fun fact: the acceptance/build-info roachtest is about twice as fast as
the old build-info acceptance test.

See #29151

Release note: None

petermattis committed:
roachtest: add acceptance/build-info

Move the build-info acceptance test to a new acceptance/build-info
roachtest. Move the existing cli/node-status to
acceptance/cli/node-status. The acceptance roachtests are all going to
share the same cluster to amortize the cluster setup/teardown time.

Fun fact: the acceptance/build-info roachtest is about twice as fast as
the old build-info acceptance test.

See #29151

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/29437)
<!-- Reviewable:end -->

petermattis committed:
roachtest: add acceptance/event-log

Move the event-log acceptance test to an acceptance/event-log roachtest.

Release note: None

petermattis commented:
Ha! The teamcity failure is for the next "local" acceptance test.

I'm going to plow through moving a few more of these. I've added another commit which migrates `TestEventLog`.

petermattis commented:
bors r=benesch
